pkg/convert/json_ngxtranslate.go

```diff
@@ -0,0 +1,72 @@
+package convert
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+var ErrWrongKey = errors.New("Wrong key type")
+
+func fromNgxTranslate(b []byte) (messages model.Messages, err error) {
```

Comment:
Shouldn't we use first letter upper-case if we want use these functions outside this file?